packages/app/src/navigation/SidebarNavigation.cy.tsx

```diff
@@ -28,11 +28,11 @@ describe('SidebarNavigation', () => {
 
   it('shows tooltips on hover', () => {
     mountComponent(false)
-    cy.get('[data-cy="sidebar-header"').realHover()
+    cy.get('[data-cy="sidebar-header"').trigger('mouseenter')
```

Comment:
Very flaky locally with `realHover`. Was debugging a failing test here and couldn't get past this step in open mode.

Reply:
I generally prefer mouseenter as well unless is doesn't work, good change.